cpp/src/arrow/json/parser.cc

```diff
@@ -55,8 +55,8 @@ static Status ParseError(T&&... t) {
 }
 
 const std::string& Kind::Name(Kind::type kind) {
-  static const std::string names[] = {"null",   "boolean", "number",
-                                      "string", "array",   "object"};
+  static const std::string names[] = {"null",  "boolean", "number",          "string",
```

Comment:
Not sure why this weird format, but this is what clang-format gives.

Reply:
This can be avoided by enclosing this between `// clang-format off` and `// clang-format on` tags, for the record.